src/main/java/pokecube/legends/tileentity/GenericBookshelfEmptyTile.java

```diff
@@ -131,7 +130,7 @@ public ActionResultType interact(final PlayerEntity player, final Hand hand, fin
             }
         }
         // place book
-        else if (!playerHand.isEmpty() && this.canPlaceItem(number, playerHand) && hand == Hand.MAIN_HAND)
+        else if (!playerHand.isEmpty() && this.canPlaceItem(number, playerHand) && hand == Hand.MAIN_HAND && !(number >= 9))
```

Comment:
this should probably use `number < 9` instead of `!(number >= 9)`, as is a bit simpler to read

Reply:
Oh good point. I don't know why I didn't think of that. :D